CHANGELOG.rst

```diff
@@ -7,6 +7,14 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 1.7.1 (2024-05-13)
+--------------------------
+
+* Fixed: require ``pandas>=2.0.1``
+  to ensure ``pyarrow`` based data types
+  work under all conditions
```

Comment:
I find `work under all conditions`  vague.  Is there a way to make it more explicit to the reader of the `CHANGELOG`?  Otherwise it could also be deleted.

Reply:
Good point, I cannot be more specific at this point. What does not work with `pandas==2.0.0` is conversion from `pyarrow[float]` to `pandas.Timedelta` values, whereas with `pandas=2.0.1` it works.

I simplified the text to:

![image](https://github.com/audeering/audb/assets/173624/d4e59cff-7d49-4554-bcc7-b5299ef82f06)

Reply:
This is ok with me. Resolving. 

I am not sure where to approve the review after having asked for changes. 

Anyway: I think thisis good to go now.